### DISCUSSION TITLE
Map View Showing My Location as Center

### DISCUSSION THREAD

**Discussion by dfreder1** [magicbug/Cloudlog#2993] [General]
My QTH is California, but the map shows the red dots for my QSO's in Japan to the right (long path), not to the left (short path). Is this a setting?
<img width="1156" alt="Screenshot 2024-02-25 at 10 12 10 PM" src="https://github.com/magicbug/Cloudlog/assets/1451568/cb48a806-94f8-4dfb-91ab-358d5483d8ba">

**Comment by magicbug** [comment]
Yes aware of this, it seems to be a limitation of the Leafletjs https://leafletjs.com that is used for mapping - I would love to fix it but currently don't know the solution

**Comment by daniel981** [comment]
I just wanted to mention that I have the exact issue. I came to the forum, searched, and saw that it was already posted, so I just wanted to add myself to the list. Also in California and it seems the map it centered on GMT and populates to either side, rather than the QTH of the user. Hopefully a fix can be found.
